test/integration_test/testspec/storagecluster/storagecluster-with-all-components.yaml

```diff
@@ -0,0 +1,19 @@
+kind: StorageCluster
+apiVersion: core.libopenstorage.org/v1
```

Comment:
how about the pvc controller? and can we add some basic metadata here? so it's still a valid object that has all the fields required, test utils should be able to update the fields.

Reply:
Added `pvc-controller` annotation to the spec